### PR TITLE
Magic UV: Pass correct argument for operator identifier

### PR DESCRIPTION
**Pull request by PratikBorhade302**
Fix the invalid argument passed in place of operator id.

### CODE REVIEW

**Review by nutti — APPROVED** [review]
LGTM

Thanks,

File: src/magic_uv/ui/IMAGE_MT_uvs.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def draw(self, context):
         ops.group = sc.muv_align_uv_snap_point_group
         ops.target = sc.muv_align_uv_snap_point_target
 
-        ops = layout.operator(MUV_OT_AlignUV_SnapToEdge, text="Snap to Edge")
+        ops = layout.operator(MUV_OT_AlignUV_SnapToEdge.bl_idname, text="Snap to Edge")
```
Comment (nutti) [inline src/magic_uv/ui/IMAGE_MT_uvs.py:121]: pep8 test is failed.
Could you apply this suggestion?

```suggestion
        ops = layout.operator(MUV_OT_AlignUV_SnapToEdge.bl_idname,
                              text="Snap to Edge")
```

**Review by nutti — CHANGES_REQUESTED** [review]
pep8 test is failed.
Could you apply my suggestion?

**Comment by nutti** [comment]
@PratikBorhade302 

Sorry, I missed pep8 test is failed.
Could you apply this suggestion?
If so, I can merge your request :)

**Comment by PratikBorhade302** [comment]
Hi, updated the change.
I think I also need to update patch submitted at [developer.blender.org](https://developer.blender.org/D13980)

**Review by nutti — APPROVED** [review]
LGTM

**Comment by nutti** [comment]
@PratikBorhade302 

Thanks for your contribution!
Your patch is now merged.

> I think I also need to update patch submitted at [developer.blender.org](https://developer.blender.org/D13980)

Yeah, it is helpful for me to update the patch.

**Pull request merged**